src/core/Akka.Remote.Tests/RemoteRouterSpec.cs

```diff
@@ -46,7 +46,7 @@ public RemoteRouterSpec()
             akka.remote.retry-gate-closed-for = 1 s #in the event of a Sys <--> System2 whoosh (both tried to connect to each other), retry quickly
             akka.actor.provider = ""Akka.Remote.RemoteActorRefProvider, Akka.Remote""
             akka.remote.helios.tcp {
-                hostname = localhost
+                hostname = 127.0.0.1
```

Comment:
Had a bug where DNS resolution wasn't performed properly on the client; changed this spec to IPs to confirm DNS was the issue. Now we have a blend of specs that use `localhost` and `127.0.0.1`. Which I think is fine - but we should have a more explicit spec that tests for DNS resolution on the default transport. I'll add an issue for that.